wire-ios/Wire-iOS Tests/ConversationMessageCell/RecentlyUsedEmojiPeristenceCoordinatorTests.swift

```diff
@@ -0,0 +1,49 @@
+
+//
+// Wire
+// Copyright (C) 2023 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+@testable import Wire
+import XCTest
+
+final class RecentlyUsedEmojiPeristenceCoordinatorTests: XCTestCase {
+
+    // MARK: setUp
+
+    override func tearDown() {
+        RecentlyUsedEmojiPeristenceCoordinator.store(RecentlyUsedEmojiSection(capacity: 15))
+        super.tearDown()
+    }
+
+    func test_RecentlyUsedEmoji_isEmpty() {
```

Comment:
The name of the test doesn't follow the same style, you use uppercase and lowercase letters after the "_".
Also the name and content of the test doesn't explain why it should be empty.
I would add:

```
// WHEN
RecentlyUsedEmojiPeristenceCoordinator.store(RecentlyUsedEmojiSection(capacity: 15))
```